js/packages/wallet-adapter-mobile/src/adapter.ts

```diff
@@ -234,16 +235,51 @@ export class SolanaMobileWalletAdapter extends BaseMessageSignerWalletAdapter {
     async sendTransaction(
         transaction: Transaction,
         connection: Connection,
-        _options?: SendOptions,
+        options?: SendOptions,
     ): Promise<TransactionSignature> {
         return await this.runWithGuard(async () => {
             const { authToken } = this.assertIsAuthorized();
             try {
                 return await this.transact(async (wallet) => {
-                    await this.performReauthorization(wallet, authToken);
+                    let targetCommitment: Finality;
+                    switch (connection.commitment) {
+                        case 'confirmed':
+                        case 'finalized':
+                        case 'processed':
+                            targetCommitment = connection.commitment;
+                            break;
+                        default:
+                            targetCommitment = 'finalized';
+                    }
```

Comment:
```suggestion
                    const targetCommitment = connection.commitment || 'finalized';
```
This works, right?

Reply:
`commitment` is a `Commitment`. I wanted to narrow it to a `Finality`.